templates/payg/algolia.mdx

```diff
@@ -2,4 +2,279 @@
+            "interval": "monthly",
+            "amount_cents": 0,
+            "amount_currency": "USD",
+            "trial_period": 0.0,
+            "pay_in_advance": false,
+            "charges": [
+                {
+                    "lago_billable_metric_id": "15879740-102e-43bd-a6bc-15d865bb1b85",
+                    "billable_metric_code": "search_requests",
+                    "charge_model": "package",
+                    "invoiceable": true,
+                    "pay_in_advance": false,
+                    "prorated": false,
+                    "min_amount_cents": 0,
+                    "properties": {
+                        "amount": "1.50",
+                        "free_units": 10000,
+                        "package_size": 1000
+                    },
+                    "filters": [],
+                    "taxes": []
+                }
+            ],
+        }
+      }
+
+
+
+  ```
+    </Tab>
+</Tabs>
+
+
+## Step 3: Offer volume and commitment discounts through subscriptions
+
+_“When committing to the equivalent of 100 Search units per month or more, you automatically unlock a discount compared to PAYGO. Please contact sales for more information.”_
+
+Algolia doesn't provide much information about their volume and commitment discounts but we know that customers who commit to making at least 100,000 Search requests per month (for Algolia, a “Search unit” is a package of 1,000 requests) can benefit from a **reduced price**.
+We are going to make some assumptions to see how we could implement this logic.
+
+
+If we want to keep our package pricing model, we should convert the commitment part of our plan into a **subscription**. 
+
+<Tabs>
+  <Tab title="Dashboard">
+    With the standard pricing, 100,000 requests would be charged \$150 (i.e. 100 packages of 1,000 requests at \$1.50 each). Since we want to offer a discount to our customers, we can create a new plan with a monthly subscription of \$120, paid in advance. Then, we set the package price for 1000 requests at \$1.20. Also, we include 100,000 free units, which are covered by the subscription fee.
+
+  <Frame>
+    <img src="/templates/images/algolia-subscription.png" />
+  </Frame>
+
+  </Tab>
+  
+  <Tab title="API">
+  ```bash Introduce a subscription
+      LAGO_URL="https://api.getlago.com"
+      API_KEY="__YOUR_API_KEY__"
+      curl --location --request POST "$LAGO_URL/api/v1/plans" \
+      --header "Authorization: Bearer $API_KEY" \
+      --header 'Content-Type: application/json' \
+      --data-raw '{
+          "plan": {
+            "name": "Algolia committed",
+            "code": "algolia_committed",
+            "interval": "monthly",
+            "amount_cents": 12000,
+            "amount_currency": "USD",
+            "trial_period": 0.0,
+            "pay_in_advance": true,
+            "charges": [
+                {
+                    "lago_billable_metric_id": "15879740-102e-43bd-a6bc-15d865bb1b85",
```

Comment:
Same comment as above. You should use the billable_metric_id

Reply:
Done ✔️